### ISSUE TITLE
[Request] Make the colors of the bubble and the dots adjustable

### ISSUE THREAD

**Issue by ErikSteiner** [martinpl/tray-icons-reloaded#42]
Currently I use [pop-gtk-theme](https://github.com/pop-os/gtk-theme), [Dash to Panel](https://extensions.gnome.org/extension/1160/dash-to-panel/) and [Night Theme Switcher](https://extensions.gnome.org/extension/2236/night-theme-switcher/).

~~My problem is, that the three dots are dark grey and hard to read. This is only a problem if the Light theme is active. As soon as I switch to the dark theme, the three dots become white.~~

Moreover the tray bubble is white as it seems like it uses the colors of the POP OS Light theme, but the icons are white, too. As soon as I switch to the dark theme, the icons are good readable: White icons with dark background.

So it would be nice to have a setting to ~~control the color of the three dots and to~~ handle the icon color. I could imagine that one would have to make the symbol color dynamic, for all those Night Theme Switcher users. Because, as soon as it gets evening, the Night Theme Switcher switches the theme from light to dark theme.

**Light Theme**
![Bildschirmfoto vom 2021-05-27 12-21-45](https://user-images.githubusercontent.com/2684018/119810342-3cfe8200-bee6-11eb-809f-87a2151da849.png)

**Dark Theme**
![Bildschirmfoto vom 2021-05-27 20-15-54](https://user-images.githubusercontent.com/2684018/119876749-8374d080-bf28-11eb-8aee-c71e83a21e71.png)

**Comment by brachiocephalic** [comment]
I agree. I can barely see the three dots, not to take away from the extension. It works great overall!

**Comment by ErikSteiner** [comment]
Thank you. I edited my request, as it seems like https://github.com/MartinPL/Tray-Icons-Reloaded/issues/28 could be the cause for the "three dots color" behavior. Currently the three dots are always white on **Dash to Panel**. Unfortunately I have no idea why it is working.. So for now my request is only related to the background of the bubble.
